coda-prototype/model.js

```diff
@@ -1542,6 +1542,123 @@ class StorageManager {
     }
 }
 StorageManager._MAX_ACTIVITY_SAVE_FREQ = 3;
+class FileIO {
```

Comment:
I'm not sure the UI should live with the model?

Reply:
Me too. But I felt it belonged near StorageManager so put it here for now, with the intention of (as mentioned) moving them both elsewhere later.